src/app/selectors.js

```diff
@@ -565,6 +565,7 @@ export const getWSControllerURL = createSelector(
 */
 export const getControllerDataByUUID = (controllerUUID) => {
   return createSelector(getControllerData, (controllerData) => {
+    if (!controllerData) return null;
```

Comment:
When I attempted this fix myself locally I ended up with intermittent failures but on your branch it seems to work as expected 🤷‍♂️ 

Lets keep our eyes out for a race here.